isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -914,6 +916,12 @@ namespace Isis {
   }
 
 
+  void Cube::write(const Table &table) {
+    Blob tableBlob = table.toBlob();
+    write(tableBlob);
+  }
```

Comment:
Similarly with Austin's PR I don't know if we need a write function for the table object specifically. Just adding `write(table.toBlob())` seems okay to me

Reply:
We (Jesse, Austin, and I) will be discussing this offline and come to a conclusion on how the writes should be handled